app/routes.py

```diff
@@ -89,7 +81,13 @@ def search_results(movies: List[Tuple[str, str]]) -> str:
     return render_template("search_results.html", movies=movies)
 
 
-@app.route("/error/<error>")
-def error_handler(error: str) -> str:
-    """Get page with error message."""
+@app.errorhandler(SQLAlchemyError)
+def handle_db_error(error: str) -> str:
+    """Get page with db error message."""
     return render_template("error.html", error=error)
+
+
+@app.errorhandler(Exception)
```

Comment:
You catch all possible exceptions.  This is a bad practice.  Just throw a spicific one and handle it here.